app/services/encrypted_key_maker.rb

```diff
@@ -59,7 +59,7 @@ def make_kms(user_access_key)
   end
 
   def unlock_kms(user_access_key, encryption_key)
-    ciphertext = user_access_key.xor(decode(encryption_key))
+    ciphertext = user_access_key.xor(decode(encryption_key)).sub(KEY_TYPE[:KMS], '')
     user_access_key.unlock(aws_client.decrypt(ciphertext_blob: ciphertext).plaintext)
```

Comment:
Would it be safer to drop the first `n` characters of the string based on `KEY_TYPE[:KMS].length`, just in case there are other occurrences of the string?